site/gatsby-config.js

```diff
@@ -211,6 +211,18 @@ module.exports = {
         sitemap: null,
         policy: [{userAgent: '*', disallow: '/v1'}]
       }
+    },
+    {
+      resolve: 'gatsby-plugin-matomo',
+      options: {
+        siteId: '831',
+        matomoUrl: 'https://webanalytics.digiaiiris.com/js',
+        siteUrl: 'https://hds.hel.fi',
+        matomoJsScript: 'piwik.min.js',
+        matomoPhpScript: 'tracker.php',
+        requireConsent: true,
```

Comment:
It seem that matomo can be used without users consent to cookies:

_By disabling tracking cookies, you may also use Matomo [without needing to display a cookie consent screen](https://matomo.org/cookie-consent-banners/). You can also [keep tracking when they reject cookie consent](https://matomo.org/faq/new-to-piwik/how-can-i-still-track-a-visitor-without-cookies-even-if-they-decline-the-cookie-consent/) by keeping cookies disabled._

Source: https://matomo.org/blog/2020/02/web-analytics-cookies-gdpr/

The tracking will become less accurate. Would it then mean that we would not have to ask cookie consent for it? And even if user denies matomo in the cookie consent, we can still track the user but just without cookies? It would feel weird that you can track cookies without having to ask consent from them?

Reply:
Sounds like a good idea 👍